executor/pipeline/fetch.rs

```diff
@@ -376,11 +378,12 @@ fn execute_list_subfetch(
             initial_row[local_row_position.as_usize()] =
                 row[parent_row_position.as_usize()].as_reference().into_owned();
         });
-        let initial_row = MaybeOwnedRow::new_owned(initial_row, row.multiplicity());
+        let initial_row = MaybeOwnedRow::new_owned(initial_row, row.multiplicity(), Provenance::INITIAL);
         Pipeline::build_read_pipeline(
             snapshot,
             thing_manager,
             variable_registry.variable_names(),
+            Arc::new(ParametrisedQueryStructure::empty()), // TODO:?
```

Comment:
todo?

Reply:
I don't know if we want query structures within `fetch`. May as well not have them.

Reply:
yeah no need